dotnet/src/VectorDataIntegrationTests/SqliteVecIntegrationTests/Support/SqliteTestEnvironment.cs

```diff
@@ -26,7 +20,7 @@ internal static bool TryLoadSqliteVec(SqliteConnection connection)
 
             try
             {
-                connection.LoadExtension(VectorSearchExtensionName);
+                connection.LoadVector();
```

Comment:
@roji I was not sure whether I should keep doing this here, or move the logic to the `SqliteCollection` class and do it there

Reply:
We already have code SqliteCollection.GetConnectionAsync which is responsible for opening the SqliteConnection and loading the extension - definitely update that to use LoadVector().

The code here is only for testing whether sqlite_vec is properly installed, and skip the tests if otherwise (so they don't fail). I think that now that we have a nuget taking care of that, the whole thing can be removed - we should just assume that sqlite_vec is properly installed, and the test should fail otherwise. So remove all the mechanism around detecting this?

On that note... I think we should now remove SqliteCollectionOptions.VectorSearchExtensionName - now that extension deployment/loading is all handled by the nuget, the user should never to configure the name of the DLL etc.